packages/fxa-content-server/app/scripts/templates/confirm_signup_code.mustache

```diff
@@ -20,7 +20,7 @@
         <p class="verification-email-message">{{#unsafeTranslate}}Please enter the verification code that was sent to %(escapedEmail)s within 5 minutes.{{/unsafeTranslate}}</p>
         <form novalidate>
             <div class="input-row">
-                <input type="number" pattern="\d*" class="tooltip-below token-code" placeholder="{{#t}}Enter 6-digit code{{/t}}" required autofocus />
+                <input type="text" inputmode="numeric" pattern="\d[ ]*" class="tooltip-below otp-code" placeholder="{{#t}}Enter 6-digit code{{/t}}" required autofocus />
```

Comment:
Using `type="text" inputmode="numeric"` allows the numberpad to be displayed on mobile devices. This also removes spaces from input.

Reply:
I find these comments really useful, is there no way to include them in the markup (as comments) or in the commit message (so git commit blame might enlighten someone)?

I know one approach is to dig up the commit, then find the PR, then find this comment, but that seems like a lot more work.

Reply:
Will add as markup.